pkg/common/k8s.go

```diff
@@ -262,3 +266,133 @@ func EnrichProbe(probe **v1.Probe, defaultProbe *v1.Probe) {
 		(*probe).FailureThreshold = defaultProbe.FailureThreshold
 	}
 }
+
+// GetStringValueFromSecret returns the string value from the secret by the given key and true if the key exists
+func GetStringValueFromSecret(secret *v1.Secret, key string) (string, bool) {
+	if secret == nil {
+		return "", false
+	}
+	value, ok := secret.Data[key]
```

Comment:
I might be wrong here, but isn't the `Data` member of `v1.Secret` contain the base64-encoded string?

Reply:
Great, so it is smarter than I expected 😄